client/extensions/woocommerce/app/order/order-fulfillment/index.js

```diff
@@ -158,12 +233,25 @@ class OrderFulfillment extends Component {
 						{ errorMessage && <Notice status="is-error" showDismiss={ false }>{ errorMessage }</Notice> }
 					</form>
 				</Dialog>
+				{ wcsEnabled && <QueryLabels orderId={ order.id } /> }
+				{ wcsEnabled && <LabelPurchaseDialog orderId={ order.id } siteId={ site.ID } /> }
 			</div>
 		);
 	}
 }
 
 export default connect(
-	undefined,
-	dispatch => bindActionCreators( { createNote, updateOrder }, dispatch )
+	( state, { order, site } ) => {
+		const labelsLoaded = wcsEnabled && Boolean( areLabelsLoaded( state, order.id, site.ID ) );
+		const hasLabelsPaymentMethod =
+			( wcsEnabled && ! labelsLoaded ) ||
```

Comment:
Under what circumstance should `wcsEnabled && ! labelsLoaded` mean hasLabelsPaymentMethod is true?